src/index.ts

```diff
@@ -1 +1,8 @@
 export * from "./commands";
+export { Settings } from "./settings";
+export {
+  MigrateLogger,
+  MigrateLogFactory,
+  MigrateLogLevel,
+  MigrateLogMeta,
+} from "./logger";
```

Comment:
Exporting here directly from @graphile/logger makes it clearer where these names are coming from without having to chase it through. We should also export the default logger so that users can pass back e.g. `logger: defaultLogger.scope({ ... })` passing their own custom scopes they want with minimal effort.

```suggestion
export { Logger, LogFunctionFactory, LogLevel } from "@graphile/logger";
export { defaultLogger } from "./logger";
```